templates/web/index.html

```diff
@@ -70,6 +71,16 @@
                         {{ task.deadline|date:"d.m.Y, H:i" }} <span class="text-muted">({{ task.deadline|timeuntil }} left)</span>
                         {% endif %}
                     </td>
+                    <td class="text-center">
+                        {% if task.accepted_submit_num %}
+                            <a href="{% url 'task_detail' login=task.student assignment_id=task.id submit_num=task.accepted_submit_num %}"
+                               title="Go to latest submit">
```

Comment:
```suggestion
                               title="Go to evaluated submit">
```
It's not necessarily the latest one.